spec/models/food_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
+
+RSpec.describe Food, type: :model do
+  describe 'Food should have valid attributes' do
+    user = User.create(name: 'Name')
+    food = Food.create(name: 'f_name', measurement_unit: 'kg', quantity: 30, price: 10, user:)
+
+    it 'should be valid with valid input fields' do
+      expect(food).to be_valid
+    end
```

Comment:
- [ ] This test is failing as shown in the image below 👇 Please try to fix it 👍 
![image](https://user-images.githubusercontent.com/46839548/202441171-db6b2054-5540-494f-affa-8eb98652e6c7.png)